code/modules/mob/living/carbon/inventory.dm

```diff
@@ -65,9 +65,11 @@
 			put_in_hands(I)
 			update_inv_hands()
 		if(slot_in_backpack)
-			if(I == get_active_held_item())
-				unEquip(I)
-			I.loc = back
+			var/obj/item/weapon/storage/B = back
+			var/prev_jimmies = B.rustle_jimmies
+			B.rustle_jimmies = FALSE //don't conspicously rustle

```

Comment:
ebows do rustle and in any case if they didn't do that I'd have to modify whatever var they used to avoid rustling so.....